src/addtask.js

```diff
@@ -0,0 +1,50 @@
+// eslint-disable-next-line import/no-cycle
+import deleteTask from './deleteTask.js';
+import addCheckboxEvents from './completed.js';
+import editDescriptions from './editDescription.js';
+
+const compare = (a, b) => {
+  if (a.index < b.index) {
+    return -1;
+  }
+  if (a.index > b.index) {
+    return 1;
+  }
+  return 0;
+};
+
+const createList = (items) => {
+  items.sort(compare);
+  let updatedTasks = '';
+  items.forEach((item) => {
+    let checked = '';
+    if (item.completed) {
+      checked = ' checked';
+    }
+    updatedTasks += `<div class="item${checked}"><input class="checkbox" value="${item.index}" type="checkbox"${checked}><input class="taskText" type="text" value="${item.description}"><i class="far fa-trash-alt"></i></div>`;
+  });
+  const list = document.getElementById('todolist');
+  list.innerHTML = updatedTasks;
+  addCheckboxEvents(items);
+  editDescriptions(items);
+  deleteTask(items);
+};
+
+function addTask(items) {
```

Comment:
- [] Please try to use the newer es6 syntax in situations where it is possible. It is recommended to use the newer arrow function syntax.
something like this
```suggestion
const addTask = (items) => {
```